homework/index.js

```diff
@@ -30,18 +15,141 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+  function renderRepositories(repository, container) {
+    container.innerHTML = '';
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, {
```

Comment:
An improvment you may apply, is changing the `td` on the hard coded text fields to be `th` as a header for what it represents.